packages/commonwealth/client/scripts/models/ChainInfo.ts

```diff
@@ -297,8 +310,11 @@ class ChainInfo {
     defaultOverview?: boolean;
     defaultPage?: DefaultPage;
     hasHomepage?: boolean;
+    cosmos_governance_version?: string;
     chain_node_id?: string;
     discord_bot_webhooks_enabled?: boolean;
+    directory_page_enabled?: boolean;
+    directory_page_option?: string;
```

Comment:
"option" is too generic. It should be like, "directory_page_chain_node_id" or something and then reference the node ("base") selected.

Also -- when + how does it get set?

Reply:
Should be set through the updateChains api when the admin makes his choice from the dropdown menu on the directory page.

If null, should be set to the chain node id of the chain that the admin is on when viewing this dropdown.